loader.js

```diff
@@ -1,26 +1,29 @@
 const { spawn } = require('child_process');
 const { join, dirname } = require('path');
+const log = require('console-emoji');
 
 function spawnWasmPack({ isDebug, cwd }) {
   const bin = 'wasm-pack';
 
   const args = [
     '--verbose',
     'build',
-    '--target', 'browser',
-    '--mode', 'no-install',
```

Comment:
I liked that formating, but I looks like you prettified it. That's fine too.

Reply:
Yes, sorry. I have prettier by default in my IDE. I can deactivate it if you want. No problem with that.

Reply:
I've kept the format you got there and remove the trailing commas.